### PR TITLE
Documentation for --parameter-file.

### PR DESCRIPTION
**Pull request by porridge**
**What this PR does / why we need it**:

Docs for https://github.com/kudobuilder/kudo/pull/1410

Signed-off-by: Marcin Owsiany <mowsiany@D2iQ.com>

### CODE REVIEW

File: content/docs/cli.md [hunk 1]
```diff
@@ -206,10 +219,16 @@ This will install new instance of Kafka Operator which maps to Kafka operator ve
...
+Use `--parameter`/`-p` and/or `--parameter-file`/`-P` for setting parameters.
+
+For example:
 
 ```bash
-$ kubectl kudo install kafka --instance=my-kafka-name --parameter ZOOKEEPER_URI=zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181 --parameter ZOOKEEPER_PATH=/small -p BROKERS_COUNTER=3
+$ cat kafka-parameters.yaml
+ZOOKEEPER_URI: zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181
+ZOOKEEPER_PATH: /small
+$ kubectl kudo install kafka --instance=my-kafka-name -p BROKERS_COUNTER=3 -P kafka-parameters.yaml
```
Comment (yankcrime) [inline content/docs/cli.md]: Might as well fix the parameter name for the Kafka operator while you're editing this line - it should be `BROKER_COUNT`

File: content/docs/cli.md [hunk 2]
```diff
@@ -206,10 +219,16 @@ This will install new instance of Kafka Operator which maps to Kafka operator ve
...
 operator.kudo.dev/kafka unchanged
 operatorversion.kudo.dev/kafka unchanged
 No instance named 'my-kafka-name' tied to this "kafka" version has been found. Do you want to create one? (Yes/no)
```
Comment (yankcrime) [inline content/docs/cli.md]: KUDO no longer prompts for this so we can remove this line from the example.

Comment (yankcrime) [reply content/docs/cli.md]: Ah, just spotted #220 !

File: content/docs/cli.md (hunk 1 above)
Comment (porridge) [inline content/docs/cli.md]:
```suggestion
$ kubectl kudo install kafka --instance=my-kafka-name -p BROKER_COUNT=3 -P kafka-parameters.yaml
```

File: content/docs/cli.md [hunk 3]
```diff
@@ -138,7 +138,20 @@ A specific application version on the official GitHub repo. (default to the most
 :::
 
 ::: flag -p, --parameter (stringArray)
-The parameter name and value separated by '='
+The parameter name and value separated by '='. See also `-P`
+:::
+
+::: flag -P, --parameter-file (stringArray)
+Path to a YAML file with parameter values. The top-level element in this file must be a mapping,
+where keys are parameter names and values are the parameter values.
```
Comment (zen-dog) [inline content/docs/cli.md:146]: I suggest an example mapping here

**Review by zen-dog — APPROVED** [review]
🚢 with a nit

**Review by yankcrime — APPROVED** [review]

File: content/docs/cli.md (hunk 3 above)
Comment (porridge) [reply content/docs/cli.md:146]: I added a pointer to the one below.

Comment (zen-dog) [reply content/docs/cli.md:146]: Wdyt about a more complex example with nested parameter values?

Comment (porridge) [reply content/docs/cli.md:146]: I'd say let's have an actual operator which can consume such parameters and then we can use it as a non-vaporware example.

**Pull request merged**